### PR TITLE
Chill 0.3

### PR DESCRIPTION
**Pull request by dhuth**

### CODE REVIEW

File: var/spack/repos/builtin/packages/chill/package.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
+##############################################################################
+# Copyright (c) 2013-2018, Lawrence Livermore National Security, LLC.
```
Comment (scheibelp) [inline var/spack/repos/builtin/packages/chill/package.py]: @dhuth could you update to the new license:

```
# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
# Spack Project Developers. See the top-level COPYRIGHT file for details.
#
# SPDX-License-Identifier: (Apache-2.0 OR MIT)
```

File: var/spack/repos/builtin/packages/rose/add_spack_compiler_recognition.patch [hunk 2]
```diff
@@ -1,13 +0,0 @@
-diff --git a/config/compiler-defs.m4 b/config/compiler-defs.m4
```
Comment (scheibelp) [inline var/spack/repos/builtin/packages/rose/add_spack_compiler_recognition.patch]: Is there a reason you are modifying patches in other packages? Could you update the PR description to explain why?

File: var/spack/repos/builtin/packages/chill/package.py [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+    depends_on("boost@1.60.0:1.69.0")
+    depends_on('iegenlib')
+    depends_on('isl')
+    depends_on('python')
+    
+    def autoreconf(self, spec, prefix):
+        bash = which('bash')
+        
+        with working_dir(self.stage.source_path + '/..'):
+            bash('git clone -b {0} {1}'.format(rose_version, rose_git, 'rose_src')
```
Comment (tgamblin) [inline var/spack/repos/builtin/packages/chill/package.py]: This needs to be converted to a dependency or a resource.

Comment (dhuth) [reply var/spack/repos/builtin/packages/chill/package.py]: I might send a new merge request with a 'rose-for-chill' package.
I'll update the license as well.

Comment (dhuth) [reply var/spack/repos/builtin/packages/chill/package.py]: new pull request: https://github.com/spack/spack/pull/11406

**Comment by tgamblin** [comment]
Superseded by #11406

**Pull request closed without merging**